http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -10,138 +10,132 @@ import org.broadinstitute.dsde.workbench.leonardo.db.{clusterQuery, followupQuer
 import org.broadinstitute.dsde.workbench.leonardo.model.google.{ClusterStatus, MachineType}
 import org.broadinstitute.dsde.workbench.leonardo.util.ClusterHelper
 import _root_.io.chrisdavenport.log4cats.Logger
-import com.typesafe.scalalogging.LazyLogging
 import org.broadinstitute.dsde.workbench.leonardo.model.google.ClusterStatus.Stopped
 import org.broadinstitute.dsde.workbench.leonardo.monitor.LeoPubsubMessage._
 import org.broadinstitute.dsde.workbench.model.WorkbenchException
 import cats.implicits._
-
 import scala.concurrent.ExecutionContext
 
 class LeoPubsubMessageSubscriber[F[_]: Async: Timer: ContextShift: Logger: Concurrent](
-  subscriber: GoogleSubscriber[IO, LeoPubsubMessage],
+  subscriber: GoogleSubscriber[F, LeoPubsubMessage],
   clusterHelper: ClusterHelper,
-  dbRef: DbReference[IO]
-)(implicit executionContext: ExecutionContext)
-    extends LazyLogging {
+  dbRef: DbReference[F]
+)(implicit executionContext: ExecutionContext) {
 
-  def messageResponder(message: LeoPubsubMessage): IO[Unit] = {
-    val response = message match {
+  def messageResponder(message: LeoPubsubMessage): F[Unit] = {
```

Comment:
it would have been super helpful if this logged whenever it received a message when debugging